src/Esfa.Vacancy.Register.Api/Mappings/RecruitApprenticeshipMapper.cs

```diff
@@ -0,0 +1,221 @@
+        {
+            var liveVacancyBaseUrl = _provideSettings.GetSetting(ApplicationSettingKeys.LiveApprenticeshipVacancyBaseUrlKey);
+
+            var description = GetVacancyDescription(liveVacancy);
+
+            var trainingType =
+                (ApiTypes.TrainingType)Enum.Parse(typeof(ApiTypes.TrainingType), liveVacancy.ProgrammeType);
+
+            var trainingCode = GetTrainingCode(trainingType, liveVacancy.ProgrammeId);
+
+            var trainingTitle = GetTrainingTitle(trainingType, trainingCode);
+
+            var qualifications = GetVacancyQualification(liveVacancy);
+
+            var skills = string.Join(",", liveVacancy.Skills);
+
+            var wageText = await GetWageText(liveVacancy.Wage, liveVacancy.StartDate);
+
+            var duration = GetDurationAsText(liveVacancy.Wage);
+
+            var wageUnit = GetWageUnit(liveVacancy.Wage.WageType);
+
+            var apprenticeship = new ApiTypes.ApprenticeshipVacancy
+            {
+                VacancyReference = liveVacancy.VacancyReference,
+                Title = liveVacancy.Title,
+                ShortDescription = liveVacancy.ShortDescription,
+                Description = description,
+                WageUnit = wageUnit,
+                WorkingWeek = liveVacancy.Wage.WorkingWeekDescription,
+                WageText = wageText,
+                WageAdditionalInformation = liveVacancy.Wage.WageAdditionalInformation,
+                HoursPerWeek = liveVacancy.Wage.WeeklyHours,
+                ExpectedDuration = duration,
+                ExpectedStartDate = liveVacancy.StartDate,
+                PostedDate = liveVacancy.LiveDate,
+                ApplicationClosingDate = liveVacancy.ClosingDate,
+                NumberOfPositions = liveVacancy.NumberOfPositions,
+                EmployerName = liveVacancy.EmployerName,
+                EmployerDescription = liveVacancy.EmployerDescription,
+                EmployerWebsite = liveVacancy.EmployerWebsiteUrl,
+                ContactName = liveVacancy.EmployerContactName,
+                ContactEmail = liveVacancy.EmployerContactEmail,
+                ContactNumber = liveVacancy.EmployerContactPhone,
+                TrainingToBeProvided = liveVacancy.TrainingDescription,
+                QualificationsRequired = qualifications,
+                SkillsRequired = skills,
+                PersonalQualities = null,
+                ImportantInformation = null,
+                ApplicationInstructions = liveVacancy.ApplicationInstructions,
+                ApplicationUrl = liveVacancy.ApplicationUrl,
+                FutureProspects = liveVacancy.OutcomeDescription,
+                ThingsToConsider = liveVacancy.ThingsToConsider,
+                IsNationwide = false,
+                SupplementaryQuestion1 = null,
+                SupplementaryQuestion2 = null,
+                VacancyUrl = $"{liveVacancyBaseUrl}/{liveVacancy.VacancyReference}",
```

Comment:
We might have an issue here. The /reference/{refno} url won;t be able to resolve V2 vacancies. Will have to use the one without reference.